gradle build failure

I just did a git clone and got many gradle build errors.  A sample:

```/Users/susancrayne/ResearchStack/ResearchStack/backbone/src/main/java/org/researchstack/backbone/model/ConsentDocument.java
Error:(2, 34) error: package android.support.annotation does not exist
/Users/susancrayne/ResearchStack/ResearchStack/backbone/src/main/java/org/researchstack/backbone/model/ConsentSection.java
Error:(4, 35) error: package com.google.gson.annotations does not exist
Error:(6, 34) error: cannot find symbol class R
/Users/susancrayne/ResearchStack/ResearchStack/backbone/src/main/java/org/researchstack/backbone/model/ConsentSignature.java
Error:(2, 34) error: package android.support.annotation does not exist
/Users/susancrayne/ResearchStack/ResearchStack/backbone/src/main/java/org/researchstack/backbone/model/DocumentProperties.java
Error:(2, 35) error: package com.google.gson.annotations does not exist
/Users/susancrayne/ResearchStack/ResearchStack/backbone/src/main/java/org/researchstack/backbone/step/ConsentDocumentStep.java
Error:(2, 34) error: cannot find symbol class R
/Users/susancrayne/ResearchStack/ResearchStack/backbone/src/main/java/org/researchstack/backbone/storage/database/sqlite/DatabaseHelper.java
Error:(5, 23) error: package com.google.gson does not exist
Error:(6, 23) error: package com.google.gson does not exist
Error:(20, 31) error: package co.touchlab.squeaky.dao does not exist
Error:(21, 37) error: package co.touchlab.squeaky.db.sqlite does not exist
```

--- a/backbone/build.gradle
+++ b/backbone/build.gradle
@@ -134,7 +134,13 @@ if (project.hasProperty("android")) { // Android libraries
     task javadoc(type: Javadoc) {
         failOnError false
         source = android.sourceSets.main.java.srcDirs
-        classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
+    }
+
+    afterEvaluate {
+        javadoc.classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
+        javadoc.classpath += files(android.libraryVariants.collect { variant ->
+            variant.javaCompile.classpath.files
+        })
     }
 } else { // Java libraries
     task sourcesJar(type: Jar, dependsOn: classes) {

--- a/skin/build.gradle
+++ b/skin/build.gradle
@@ -124,6 +124,13 @@ if (project.hasProperty("android")) { // Android libraries
         source = android.sourceSets.main.java.srcDirs
         classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
     }
+
+    afterEvaluate {
+        javadoc.classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
+        javadoc.classpath += files(android.libraryVariants.collect { variant ->
+            variant.javaCompile.classpath.files
+        })
+    }
 } else { // Java libraries
     task sourcesJar(type: Jar, dependsOn: classes) {
         classifier = 'sources'

